### ISSUE TITLE
"export { default } from" not working

### ISSUE THREAD

**Issue by samiamorwas** [doczjs/docz#794]
## Bug Report

**Describe the bug**

I have several index files that re-export the default value from another file:
```js
export { default } from "./Button";
```

All of these cause the same error on build:
```
Module parse failed: Unexpected token (3:11)
You may need an appropriate loader to handle this file type.
| export { default } from './Button';
|
> if (typeof default !== 'undefined' && default && default === Object(default)) {
|   Object.defineProperty(default, '__filemeta', {
|     enumerable: true,
```

Any other type of export seems to work, but would involve changing a lot of files.

**Environment**

* OS: OSX 10.14.4
* Node/npm version: Node v8.15.1, npm 6.4.1

**Comment by jchadwick** [comment]
Is this possibly the same as or related to #790?

**Comment by fi3ework** [comment]
This bug is supposed to be fixed in 1.0.1. Have a try. @samiamorwas

**Comment by samiamorwas** [comment]
Yeah, I thought this was the same issue as #790 , but it appears that that only works for exports in the form `export { default as Something } from "./Something"`, and not for `export { default } from "./Something"`.

Digging into the fix for #790 and the code for babel-plugin-export-metadata (which I totally don't understand very well, so I could be way off base here), it looks like `export { default} from "./Something"` specifiers wind up with "local.name" and "exported.name" both being "default", which seems to be what triggers the issue. Again, I could be missing something here.

**Comment by fi3ework** [comment]
I believe it is a bug of babel-plugin-export-metadata. The workaround (#794) for #790 just considered `export { default as sth}`. I'll have a try for a better babel-plugin-export-metadata that consider different export condition.
